http-server/src/main/java/org/triplea/server/error/reporting/ErrorReportController.java

```diff
@@ -49,4 +55,16 @@ private String extractClientIp(final HttpServletRequest request) {
 
     return request.getRemoteHost();
   }
+
+  /**
+   * Checks if the user has hit their rate limit for submitting error reports.
+   *
+   * @return True if the user can submit an error report, false if they
+   *         have hit their limit.
+   */
+  @GET
```

Comment:
This endpoint is used to before we show the upload dialog. The idea is it's bad to let the user do work, like fill in optional error report details to click submit and then get an error. Instead of that we'll show a "you are rate limited dialog" instead of showing the upload error report form.